tests/unit_test.R

```diff
@@ -0,0 +1,38 @@
+library(kaiaulu)
+library(testthat)
+
+test_that("calling parse_gitlog with correct perceval path and an exist local path of github project", {
+  git_repo_path <- "~/Documents/GitHub/thrift/.git"
```

Comment:
This does not follow the agreed convention on Kaiaulu where rawdata is. Maybe check with @leilani-reich on this.

Reply:
Please apply the fixes to the subsequent tests too.